gradle.properties

```diff
@@ -3,10 +3,10 @@ dockerComposeStopContainers=true
 
 kotlinVersion=1.4.10
 okhttp3Version=4.9.+
-radarJerseyVersion=0.4.3
+radarJerseyVersion=0.6.0
```

Comment:
Has been updated to https://github.com/RADAR-base/radar-jersey/releases/tag/v0.6.2